Wireframe/index.html

```diff
@@ -9,24 +9,46 @@
   <body>
     <header>
       <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
+      <p id = "welcome">
+        Welcome to the world of Wireframes,README files and Git. You'll be an expert soon!
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="https://i.pinimg.com/736x/e0/63/fb/e063fb417771f10dbe9d3298a4178756.jpg" alt="Image of a man smilling in the foreground with three people in the background" />
```

Comment:
I like your photo 🙂 however, I think for the purposes of this exercise, it would be better to have an image that relates to a 'wireframe'.